src/shogun/machine/gp/FITCInferenceBase.cpp

```diff
@@ -109,7 +106,7 @@ void CFITCInferenceBase::init()
 	SG_ADD(&m_Sigma, "Sigma", "covariance matrix of the approximation to the posterior", MS_NOT_AVAILABLE);
 	SG_ADD(&m_ktrtr_diag, "ktrtr_diag", "diagonal elements of kernel matrix m_ktrtr", MS_NOT_AVAILABLE);
 
-	m_inducing_features=NULL;
+	m_inducing_features=SGMatrix<float64_t>();
```

Comment:
Do we start like this?
Not rather a random subset of the training data? Or is this done later?